### ISSUE TITLE
The media could not be loaded, either because the server or network failed or because the format is not supported.

### ISSUE THREAD

**Issue by pagiraud** [YunoHost-Apps/invidious_ynh#166]
Hi,

### Describe the bug

For a few weeks, more and more videos have refused to play (all videos now) and display the error "The media could not be loaded, either because the server or network failed or because the format is not supported." The rest of the interface, including the comments, are displayed.

### Context

It is [a documented bug](https://github.com/iv-org/invidious/issues/4947) on upstream Invidious, that has been marked as resolved. I thought the latest upgrade (2024.09.28-53e8a5d6) would resolve the issue for Yunohost users, but no (at least for me). As far as I understand, installation process has changed, so the Yunohost install script should reflect that. I’ve not seen anything in the pull requests that indicate such a change, but maybe I’m wrong.

Do you experience the same issue? If yes, have you an idea on how to solve it?

**Comment by tcsenpai** [comment]
Try to take a look at https://github.com/iv-org/invidious/issues/4947#issuecomment-2381565306

TLDR: Move the instance to another server if you can (else I'd try changing the container's names), use gluetun if possible and in that case configure sig_helper to use gluetun too

**Comment by tcsenpai** [comment]
Hi @pagiraud 

I'm not running it on yunohost, the steps I took were made on a plain Debian machine (running casaos), but could apply to any system as I don't think it's a platform specific issue.

**Comment by isAAAc** [comment]
> Move the instance to another server

that's not solving the problem, it's getting round it. 
How can we make it work like it used to? 

my instance is on my server, telling us to change instance is nonsense, even if it's good will, it doesn't solve anything at all ;)

**Comment by pagiraud** [comment]
I still have this problem, making Invidious totally useless, but it seems nobody else is complaining. Does anyone else have this issue?

**Comment by isAAAc** [comment]
@pagiraud 
yes, 
i removed invidious , and boycott youtube on my side, 
i can live without yt, and i use peertube instead
